src/earn/EarnPoolInfoScreen.tsx

```diff
@@ -198,11 +198,16 @@ function DepositAndEarningsCard({
           return acc.plus(
             new BigNumber(item.amount)
               .multipliedBy(tokenInfo?.priceUsd ?? 0)
-              .dividedBy(tokenInfo?.priceUsd ?? 1)
+              .multipliedBy(localCurrencyExchangeRate ?? 0)
```

Comment:
do this to convert from usd amount to local currency amount

Reply:
instead of doing this calculation twice in fiat and crypto, could we do it once in crypto and use the wrapper `useDollarsToLocalAmount` to covert this (or I think we can also just use the crypto amount with TokenDisplay to display the fiat amount)